plans/pea.tf

```diff
@@ -0,0 +1,60 @@
+resource "azurerm_resource_group" "pea" {
+  name     = "${var.prefix}pea"
+  location = "${var.location}"
+  tags {
+    env = "${var.prefix}"
+  }
+}
+
+resource "azurerm_storage_account" "pea" {
+    name                = "${var.prefix}pea"
+    resource_group_name = "${azurerm_resource_group.pea.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
+    depends_on          = ["azurerm_resource_group.pea"]
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_container_service" "pea" {
+  depends_on             = ["azurerm_resource_group.pea"]
+  name                   = "${var.prefix}pea"
+  location               = "${azurerm_resource_group.pea.location}"
+  resource_group_name    = "${azurerm_resource_group.pea.name}"
+  orchestration_platform = "Kubernetes"
+
+  master_profile {
+    count      = 1
+    dns_prefix = "${var.prefix}pea"
```

Comment:
I don't see anything here about the Master size? We need a master with a bigger disk to avoid some of the issues we have had thus far